todocore/src/main/java/com/knighttodo/todocore/service/privatedb/repository/DayTodoRepository.java

```diff
@@ -19,9 +19,12 @@ public interface DayTodoRepository extends JpaRepository<DayTodo, UUID> {
     List<DayTodo> findAllAlive();
 
     @Query("select dt from DayTodo dt where dt.day.id=:dayId and dt.removed=false and dt.day.removed=false")
-    List<DayTodo> findByDayIdAlive(@Param("dayId") UUID dayId);
+    List<DayTodo> findAllWithOrderNumber(@Param("dayId") UUID dayId);
 
     @Modifying
     @Query("update DayTodo td set td.removed=true where td.id=:dayTodoId")
     void softDeleteById(@Param("dayTodoId") UUID dayTodoId);
+
+
+
```

Comment:
remove it please, there is no body so we don't need empty space here

Reply:
Done.